src/frontend/src/pages/PPQPage/PPQPage.tsx

```diff
@@ -1,62 +1,70 @@
 
 import ppqImg from '../../assets/ppq_homepage.svg';
+import piaImg from '../../assets/pia_intake.svg';
 
 import { Link } from 'react-router-dom';
 import messages from './messages';
 
-function PPQLandingPage() {
-  // TODO: replace with actual role check
-  const isMPO = true;
-
-  return isMPO ? (
+interface IComponentProps {
+  showMPOContents: boolean;
+}
+function PPQLandingPage(props: IComponentProps) {
+  const { showMPOContents } = props;
+  return showMPOContents ? (
     <div className="bcgovPageContainer background">
-      <div className="get-started-section results-wrapper">
-        <div className="row">
-          <div className="col-md-6">
-            <h1>{messages.PIAIntakeHeading.en}</h1>
-            <br />
-            <p>{messages.PIAIntakeDescriptionText.en}</p>
-            <p>{messages.PIAIntakeHelpDescriptionText.en}</p>
+      <div>
+        <div className="mt-0">
+          <p>
+            <a href="/pia-list"> List of PIAs </a>
+            <FontAwesomeIcon className="icon" icon={faAngleRight} /> Create New
+          </p>
+        </div>
+        <div>
+          <h1 className="mb-4">Create New </h1>
+        </div>
+        <div className="container ">
+          <div className="row mt=2 mb-5 get-start-container">
+            <div className="col ">
+              <h2>
+                <b>{messages.PIAIntakeHeading.en}</b>
+              </h2>
+              <br />
+              <p>{messages.PIAIntakeDescriptionText.en}</p>
 
-            <div data-cy="ppq-btn">
-              <Link
-                to="/pia-intake"
-                className="bcgovbtn bcgovbtn__primary ppq-btn"
-              >
-                Get started
-                <FontAwesomeIcon className="icon" icon={faChevronRight} />
-              </Link>
+              <div data-cy="ppq-btn">
+                <Link
+                  to="/pia-intake"
+                  className="bcgovbtn bcgovbtn__primary ppq-btn"
+                >
+                  Start PIA Intake
+                  <FontAwesomeIcon className="icon" icon={faChevronRight} />
+                </Link>
+              </div>
+              <br />
+              <span>
+                <b>Estimated time:</b> 20 minutes
+              </span>
+            </div>
+            <div className="col col-sm-4 rounded float-end">
+              <img src={piaImg} alt="Fill form image" />
             </div>
-            <br />
-            <span>
-              <b>Estimated time:</b> 20 minutes
-            </span>
           </div>
-          <div data-cy="ppq-img" className="col-md-6 ppq-svg ">
-            <div className="get-started-content">
-              <p>
-                <FontAwesomeIcon
-                  className="icon ppq-home-warning-icon"
-                  icon={faExclamationTriangle}
-                />
-                {messages.PPQHeadingWarning.en}
-              </p>
+
+          <div data-cy="ppq-img" className="row mt=2 mb-5 get-start-container ">
```

Comment:
We should be trying to avoid classes like 'get-start-container' as they are too specific and create confusion should you have to use them elsewhere.